tests/lit-tests/float16_uniform.ispc

```diff
@@ -0,0 +1,87 @@
+//; RUN: %{ispc} %s --target=host --emit-llvm-text -O0 -o - | FileCheck %s
+
+// Tests float16 constant parsing in *e*f16 form.
```

Comment:
typo: `*e*f16` -> `*.*f16`

Reply:
That's nota  type. I amusing exponential format for f16 constant here.